api/fixtures/example.go

```diff
@@ -408,6 +410,84 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 		}
 		services = getIngressServicePublishingStrategyMapping(o.NetworkType)
 
+	case o.PowerVS != nil:
+		buildIBMCloudCreds := func(name, apikey string) *corev1.Secret {
+			return &corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Namespace: namespace.Name,
+					Name:      name,
+				},
+				Data: map[string][]byte{
+					"ibm-credentials.env": []byte(fmt.Sprintf(`IBMCLOUD_AUTH_TYPE=iam
+ IBMCLOUD_APIKEY=%s
+ IBMCLOUD_AUTH_URL=https://iam.cloud.ibm.com
+ `, apikey)),
+				},
+			}
+		}
+		powerVSResources := &ExamplePowerVSResources{
+			buildIBMCloudCreds(o.Name+"-cloud-ctrl-creds", o.PowerVS.ApiKey),
```

Comment:
I'd expect this to be granular perms for each component?

Reply:
Yes, this is something we need to explore as a overall approach how we want to address for the PowerVS flow, may be reuse the some of the work we have done for the [CCO](https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md#ibmcloud), wondering if can merge this code for now adding TODO here and start working on that once we have all the code merged for the PowerVS?